projects/plugins/jetpack/extensions/blocks/blogging-prompts/settings.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Blogging prompt settings.
+ *
+ * @since $$next-version$$
+ *
+ * @package automattic/jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\BloggingPrompts\Settings;
+
+use Automattic\Jetpack\Status\Host;
+
+/**
+ * Renders the settings field for enabling/disabling blogging prompts in the editor.
+ *
+ * @return void
+ */
+function enabled_field_callback() {
+	$option  = jetpack_are_blogging_prompts_enabled();
+	$checked = ( $option ? 'checked' : '' ); ?>
+<input name="jetpack_blogging_prompts_enabled" id="jetpack_blogging_prompts_enabled" type="checkbox" value="1"<?php echo esc_attr( $checked ); ?> />
+<p id="jetpack-blogging-prompts-enabled-description" class="description"><?php esc_html_e( 'Displays a writing prompt when starting a new post.', 'jetpack' ); ?></p>
+	<?php
+}
+
+/**
+ * Initialize the settings for blogging prompts.
+ *
+ * @return void
+ */
+function init() {
+	$host = new Host();
+
+	if ( ! $host->is_wpcom_platform() ) {
+		return;
+	}
+
+	register_setting(
```

Comment:
One potential issue with using a setting in wp-admin is that it looks like the setting gets saved in its current state whenever the page is saved. So if a user was ineligible for writing prompts, the setting would default to disabled. If they saved the settings page, then later became eligible for writing prompts, then the setting would remain disabled and they wouldn't see prompts.

🤷 I don't actually think it's a problem but just talking through edge cases.

Reply:
I'd probably suggest we address this later.